src/components/itemPage/Item.jsx

```diff
@@ -24,7 +24,8 @@ class Item extends Component {
     super();
     this.state = {
       attributeInputValue:"",
-      item:{topic:[]}
+      item:{topic:[]},
```

Comment:
A small tip to avoid changing an unrelated line: adding the new element not as the last one of the list.